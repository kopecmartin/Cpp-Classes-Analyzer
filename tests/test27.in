class A {
    short int ** var = 4;
    int *method(A & var, bool y);
}