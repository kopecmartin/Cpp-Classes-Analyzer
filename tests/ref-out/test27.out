<?xml version="1.0" encoding="utf-8"?>
<model>
    <class kind="concrete" name="A">
        <private>
            <attributes>
                <attribute name="var" type="short int **" scope="instance" />
            </attributes> 
            <methods>
                <method name="method" type="int *" scope="instance">
                    <arguments>
                        <argument name="var" type="A &amp;" />
                        <argument name="y" type="bool" />
                    </arguments>
                </method>
            </methods>
        </private>
    </class>
</model>